doc/admin-guide/plugins/multiplexer.en.rst

```diff
@@ -22,31 +22,37 @@
 |Name|
 **********
 
-|Name| is a remap plug-in that allows a request to be multiplexed one or more times and sent to
-different remap entries. Both headers and body (in case of POST or PUT methods, only) are copied
-into the new requests. Optionally POST/PUT requests can be skipped via
+|Name| is a remap plug-in that allows requests to certain origins to be multiplexed (i.e.,
+duplicated and dispatched in parallel) to one or more other hosts.  The headers are copied into the
+new requests as well as POST bodies.  Optionally POST and PUT requests can be skipped via
 ``pparam=proxy.config.multiplexer.skip_post_put=1``.
 
 Description
 ===========
 
-Actions:
+|Name| does the following for requests it is configured to multiplex:
 
-#. Adds ``X-Multiplexer: original`` header into client's request.
-#. Copies client's request (bodies are copied by transforming the request)
-#. Changes ``Host`` header of the copy according to ``pparam`` from the remap rule.
-#. Changes ``X-Multiplexer header`` to "copy".
-#. Sends the copied request with :code:`TSHttpConnect`.
+#. Add the ``X-Multiplexer: original`` header into the client's request.
+#. For each host in ``pparam`` of the remap rule:
 
-|Name| dispatches the request in background without blocking the original request. Multiplexed
-responses are drained and discarded.
+   #. Copies the resulting request (POST bodies are copied via the :ref:`HTTP Transform <developer-plugins-http-transformations>` mechanism).
+   #. Changes the ``Host`` header of the copy according to ``pparam`` from the remap rule.
+   #. Changes ``X-Multiplexer`` header value to ``copy`` instead of ``original``.
+   #. Asynchronously sends the copied request with :c:func:`TSHttpConnect`.
 
-A global timeout can be overwritten through ``multiplexer__timeout`` environment variable representing how many nanoseconds to wait. A default 1s timeout is hard-coded.
+|Name| dispatches the requests in the background without blocking the original request. Multiplexed
+responses are drained and discarded. Note that these multiplexed requests are made using
+:c:func:`TSHttpConnect` and are thus HTTP (not HTTPS) connections.
```

Comment:
@SolidWallOfCode : can you please verify I'm not mistaken here about multiplexer not being able to do HTTPS requests? Judging from the code, it simply does an TSHttpConnect and sends data and thus can't do HTTPS. But let me know if I'm missing something.